### PR TITLE
Get rid of xunit serialization warnings

### PR DESCRIPTION
**Pull request by sungam3r**
изображение
The same locally
изображение

### CODE REVIEW

**Comment by sungam3r** [comment]
Before
изображение

Tests were not discovered before actual run.

**Comment by sungam3r** [comment]
See `Test solution [Debug] without code coverage` to compare: [before](https://github.com/graphql-dotnet/graphql-dotnet/actions/runs/3851283698/jobs/6562294910#step:10:1) and [after](https://github.com/graphql-dotnet/graphql-dotnet/actions/runs/3854428714/jobs/6568410844#step:10:222). Note - there are some issues left with other cases with non-serializable arguments.

**Comment by sungam3r** [comment]
Rel: https://github.com/xunit/xunit/issues/2640

File: src/GraphQL.Tests/ValueConverterFacts.cs [hunk 1]
```diff
@@ -117,7 +117,7 @@ public void StringConversionToDateTimeOffset()
         actual.ShouldBe(expected);
     }
 
-    [Theory]
+    [TheoryEx]
```
Comment (sungam3r) [inline src/GraphQL.Tests/ValueConverterFacts.cs:120]: I changed attribute only for those tests that produced xUnit warning.

**Comment by sungam3r** [comment]
Clean output https://github.com/graphql-dotnet/graphql-dotnet/actions/runs/3855438385/jobs/6570487109#step:10:1  - 293 lines after fixing warnings, 645 before.

**Review by Shane32 — APPROVED** [review]

**Pull request merged**